src/main/resources/db/migration/V5__ClientAttributeTypeModifications.sql

```diff
@@ -0,0 +1,3 @@
+alter table client_attribute_type
```

Comment:
If this is an initial commit for DB objects, just add this to the initial definition of this table.

Reply:
Ok, fixed this.